source/Tests/IntegrationTests.cs

```diff
@@ -38,24 +41,21 @@ public void TearDown()
 
         public void GivenABunchOfPackageReferences()
         {
-            MsBuildPackageReference CreatePackageReference(string packageId, string version)
-                => new MsBuildPackageReference()
+            var xmlDoc = new XmlDocument();
```

Comment:
Getting the test data from the referenced packages for the project. This ensures they match and the packages are restored and available to the test.